CMSgov/cmcs-eregulations
# EREGCSC-1344 - About page copy updates

Resolves [EREGCSC-1344](https://jiraent.cms.gov/browse/EREGCSC-1344)

**Description**
This PR adds updated copy text and links to the about page.

**This pull request changes.**

- New copy text and links to about page per AC on JIRA story.

**Steps to manually verify this change**

1. Visit [/about](https://kubm1ucex5.execute-api.us-east-1.amazonaws.com/dev-426/about) and make sure the copy text and links match the text from the AC on the JIRA ticket.
2. In the "Criteria for Resource Links" section, visit the "detailed set of criteria" link and make sure the PDF loads.
